README.md

```diff
@@ -41,6 +41,18 @@ profile::accounts:::skel_archives:
 | `profile::cvmfs::client::quota_limit`            | Integer       | Instance local cache directory soft quota (MB) | 4096                                                                 |
 | `profile::cvmfs::client::repositories`           | Array[String] | List of CVMFS repositories to mount            | `['cvmfs-config.computecanada.ca', 'soft.computecanada.ca']`         |
 | `profile::cvmfs::client::lmod_default_modules`   | Array[String] | List of lmod default modules                   | `['gentoo/2020', 'imkl/2020.1.217', 'gcc/9.3.0', 'openmpi/4.0.3']` |
+| `profile::cvmfs::local_cvmfs_user`               | String  | Dictionary of a default user to be created locally in the cvmfs      | `cvmfs`|
+
+### profile::cvmfs::local_cvmfs_user
+
+A default local user `cvmfs-user`, for `cvmfs`
+```
+profile::cvmfs::local_cvmfs_user
```

Comment:
Given how the class is currently defined, the parameters would have to be provided like this:
```
profile::cvmfs::local_cvmfs_user::cvmfs_user: "name"
profile::cvmfs::local_cvmfs_user::uid: uid_default_value
profile::cvmfs::local_cvmfs_user::gid: gid_default_value
profile::cvmfs::local_cvmfs_user::groups: groups_default_value
```

Reply:
Done, maybe that's why it was not added, I wasn't providing the right info
thank you